lib/ansible/module_utils/aci.py

```diff
@@ -128,7 +137,9 @@ def define_method(self):
 
         # Handle deprecated method/action parameter
         if self.params['method']:
-            self.module.deprecate("Parameter 'method' or 'action' is deprecated, please use 'state' instead", '2.6')
+            # Deprecate only if state was a valid option (not for aci_rest)
+            if self.module.argument_spec('state', False):
```

Comment:
I don't think we need that complexity. Fact is that all modules will be using `state` except aci_rest which will be using `method`. So we only care about deprecating `method` if the module accepts `state`.